install_config/install/host_preparation.adoc

```diff
@@ -668,11 +666,68 @@ total 2.6M
 -rw-r--r--. 1 root root 1.3K Nov 24 00:12 hostconfig.json
 drwx------. 2 root root    6 Nov 24 00:12 secrets
 ----
-====
 
 See Docker's documentation for additional information on how to
-link:https://docs.docker.com/engine/admin/logging/overview/#/options[Configure
-Logging Drivers].
+link:https://docs.docker.com/engine/admin/logging/overview/#/options[Configure Logging Drivers].
+
+[[blocking-local-volume-usage]]
+=== Blocking Local Volume Usage
+
+When a volume is provisioned via the `VOLUME` instruction in a *_Dockerfile_* or
+via `docker run -v <volumename>`, a host's storage space is used. This could
+lead to an unexpected out of space issue, which could bring down the host.
+
+In {product-title}, users trying to run their own images risk filling the entire
+storage space on a node host. One solution to this is to deny users from running
+images with volumes. This way, the only storage a user has access to can be
+limited and the cluster administrator can assign storage quota.
+
+Using *docker-novolume-plugin* solves this issue by disallowing starting a
+container with local volumes defined. In particular, the plug-in will block
+`docker run` with:
+
+- `--volumes-from`,
+- images that have `VOLUME`(s) defined,
+- volumes early provisioned with `docker volume` command.
+
+The only thing allowed will be just bind mounts.
+
+To enable *docker-novolume-plugin*, perform the following steps on each node
+host:
+
+. Install the *docker-novolume-plugin* package:
++
+----
+$ yum install docker-novolume-plugin
+----
+
+. Enable and start the *docker-novolume-plugin* service:
++
+----
+$ systemctl enable docker-novolume-plugin
+$ systemctl start docker-novolume-plugin
+----
+
+. Edit the *_/etc/sysconfig/docker_* file and append the following to the
+`OPTIONS` list:
++
+----
+--authorization-plugin=docker-novolume-plugin
+----
+
+. Restart the *docker* service:
++
+----
+$ systemctl restart docker
+----
+
+When this plug-in is enabled, containers with local volumes defined should fail
```

Comment:
s/When this plug-in is enabled,/After you enable this plug-in,

remove "should"